README.md

```diff
@@ -286,7 +286,7 @@ or rule violations resolutions in the context of the repository.
 
 A single file or a directory with multiple files containing configurations to set provenance-specific path excludes and
 license finding curations for dependency packages to address issues found within a scan result. The `helper-cli`'s
-[GeneratePackageConfigurationsCommand](./helper-cli/src/main/kotlin/commands/GeneratePackageConfigurationsCommand.kt)
+[package-configuration create command](./helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt)
```

Comment:
Nit: Maybe write `*create*` to make the command name stand out in italics?

Reply:
The whole sub-command is `package-configuration create`. I'll use italics for this.

Reply:
Looks like exactly the same things was done in https://github.com/oss-review-toolkit/ort/pull/4190 now by @fviernau.

Reply:
Oh sorry, I hadn't seen this one early enough.